### PR TITLE
Remove redundant files

### PR DESCRIPTION
**Pull request by PhilETaylor**
Forked from #20616

### Summary of Changes

Remove files that Joomla 3.8.8 doesnt actually use, that contain minor security issues, and that generate PHP warnings when actually loaded as a template. They look abandoned (unless someone can prove to me they are not, and are actually used somewhere)

### Testing Instructions

Apply PR, Fully test com_media with doc, folder and videos in your media folders. 

### Expected result

Nothing breaks.

### CODE REVIEW

**Comment by infograf768** [comment]
Did not test if they are used or not, but don't they have to be added in script.php https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_admin/script.php#L430 ?

**Comment by mbabker** [comment]
Truthfully I'm not fully concerned about it.  Since introducing https://github.com/joomla/joomla-cms/blob/staging/build/deleted_file_check.php to the repo it's been a habit to run that when prepping each release.  So even if not included in this PR, at worst it'll be included in the "prep RC" commit.

**Comment by brianteeman** [comment]
as long as that "habit" has been added to the documentation for creating a release ;)

**Comment by mbabker** [comment]
We have documentation for critical workflows and procedures!?

**Comment by brianteeman** [comment]
Relevant history
https://github.com/joomla/joomla-cms/pull/13519#issuecomment-290026633

**Comment by PhilETaylor** [comment]
So it would be good to add to the script.php list too then right? I can do that

**Comment by PhilETaylor** [comment]
ok I give up... @mbabker something wrong with the merge, this is why I should not code in github interface and should wait until I get home! Something so simple has been upset by something you did with a revert/resync?

**Comment by PhilETaylor** [comment]
Merge conflicts after @mbabker reset/resync/revert now resolved, so that this can merge with the new staging.

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -2001,6 +2001,14 @@ public function deleteUnexistingFiles()
...
 			'/libraries/fof/view.json.php',
 			'/libraries/fof/view.php',
+
+			/**
+			 * Missed from earlier versions
+			 */
+			'/administrator/components/com_media/views/medialist/tmpl/details_doc.php',
+			'/administrator/components/com_media/views/medialist/tmpl/details_folder.php',
+			'/administrator/components/com_media/views/medialist/tmpl/details_img.php',
+			'/administrator/components/com_media/views/medialist/tmpl/details_video.php'
```
Comment (Quy) [inline administrator/components/com_admin/script.php]: Add trailing comma per c/s.

Comment (PhilETaylor) [reply administrator/components/com_admin/script.php]: grrr

**Comment by PhilETaylor** [comment]
No idea why this has not been merged in over a year, but now it reports a conflict on the files it tries to delete - even though those files still exist https://github.com/joomla/joomla-cms/tree/staging/administrator/components/com_media/views/medialist/tmpl

**Pull request closed without merging**